repartitions/models.py

```diff
@@ -0,0 +1,68 @@
+from django.db import models
+from authentication.models import User
+
+
+class Campaign(models.Model):
+    STATUS = ("CLOSED", "OPEN", "RESULTS")
+
+    id = models.AutoField(primary_key=True)
+
+    name = models.CharField(max_length=200)
+    status = models.CharField(max_length=200, default="CLOSED")
```

Comment:
choices?

Reply:
This is a one-to-many relationship so the ownership is defined on the `Choice` side

Reply:
I mean, use `models.CharField(…, choices=STATUS)`